code/game/machinery/firealarm.dm

```diff
@@ -168,8 +172,7 @@
 	obj_flags |= EMAGGED
 	update_appearance()
 	if(user)
-		user.visible_message(span_warning("Sparks fly out of [src]!"),
-							span_notice("You override [src], disabling the speaker."))
+		user.balloon_alert(user, "speaker disabled!")
```

Comment:
this makes it so that emagging is no longer obvious to observers

Reply:
okay, I'll put it back